packages/graphql-yoga/package.json

```diff
@@ -72,6 +72,7 @@
     "graphql": "^16.0.1",
     "graphql-http": "^1.7.2",
     "graphql-scalars": "1.20.1",
+    "graphql-sse": "^2.0.0",
```

Comment:
shouldn't this be a plugin?

Reply:
This is actually different than @enisdenjo 's PR. 
This makes Yoga's default SSE compatible with the client of `graphql-sse` only for non-single connection mode.
@enisdenjo 's PR allows you to use single connection mode, too.
Correct me if I'm wrong @enisdenjo ?

Reply:
But we are adding a dep to core is there way to avoid this?

Reply:
It is a dev dependency actually.